rabl.gemspec

```diff
@@ -13,10 +13,9 @@ Gem::Specification.new do |s|
   s.description = %q{General ruby templating with json, bson, xml and msgpack support}
   s.license     = 'MIT'
 
-  s.files         = `git ls-files`.split("\n")
+  s.files = Dir['lib/**/*', 'Rakefile', 'MIT-LICENSE', '*.md']
```

Comment:
can you please preserve spacing after `=` and use

```rb
`git ls-files -z -- {MIT-LICENSE,*.md,Rakefile,lib}`.split("\x0").sort
```

?

This will exclude folders and only list files

```
> pp `git ls-files -z -- {MIT-LICENSE,*.md,Rakefile,lib}`.split("\x0").sort
["CHANGELOG.md",
 "CONTRIBUTING.md",
 "MIT-LICENSE",
 "README.md",
 "Rakefile",
 "lib/rabl.rb",
 "lib/rabl/builder.rb",
 "lib/rabl/cache_engine.rb",
 "lib/rabl/configuration.rb",
 "lib/rabl/digestor.rb",
 "lib/rabl/digestor/rails3.rb",
 "lib/rabl/digestor/rails41.rb",
 "lib/rabl/digestor/rails5.rb",
 "lib/rabl/engine.rb",
 "lib/rabl/helpers.rb",
 "lib/rabl/multi_builder.rb",
 "lib/rabl/partials.rb",
 "lib/rabl/railtie.rb",
 "lib/rabl/renderer.rb",
 "lib/rabl/sources.rb",
 "lib/rabl/template.rb",
 "lib/rabl/tracker.rb",
 "lib/rabl/version.rb"]

> pp Dir['lib/**/*', 'Rakefile', 'MIT-LICENSE', '*.md'].sort
["CHANGELOG.md",
 "CONTRIBUTING.md",
 "MIT-LICENSE",
 "README.md",
 "Rakefile",
 "lib/rabl",
 "lib/rabl.rb",
 "lib/rabl/builder.rb",
 "lib/rabl/cache_engine.rb",
 "lib/rabl/configuration.rb",
 "lib/rabl/digestor",
 "lib/rabl/digestor.rb",
 "lib/rabl/digestor/rails3.rb",
 "lib/rabl/digestor/rails41.rb",
 "lib/rabl/digestor/rails5.rb",
 "lib/rabl/engine.rb",
 "lib/rabl/helpers.rb",
 "lib/rabl/multi_builder.rb",
 "lib/rabl/partials.rb",
 "lib/rabl/railtie.rb",
 "lib/rabl/renderer.rb",
 "lib/rabl/sources.rb",
 "lib/rabl/template.rb",
 "lib/rabl/tracker.rb",
 "lib/rabl/version.rb"]
```

Reply:
Done, thanks!